### DISCUSSION TITLE
Allow saved issue queries

### DISCUSSION THREAD

**Discussion by eamodio** [community/community#3008] [Mobile]
I desperately want this (per-repo) on github.com as well, but imo it is even more important in the mobile app to be able to work through issues. On the web, I end up having specific tabs for dealing with different sets of issues -- which I can't do in the mobile app.

**Comment by eamodio** [comment]
I also additionally want to be able to change the sorting of the issues and ideally by recently updated -- excluding updates made by me (which sadly you can't do on the web either).

**Comment by becca** [comment]
Is this feedback about Discussions or how you use Issues in the mobile app right now? If it's the latter, I would recommend changing the category in which this Discussions currently exists so the right product team can 👀 your feedback/request.

**Reply by eamodio** [reply]
This is feedback for the GitHub Mobile app -- managing issues on it, as searching for issues by labels, etc is extremely painful in the mobile app. And I would like to have preset (i.e. saved) queries that can be use to switching between different issue filtering/sorting. I mentioned that on the web I can use separate tabs or bookmarks to save different issue queries, but that isn't possible with the app.

**Reply by eamodio** [reply]
Now I see that it was in the wrong category, but I am sure I picked the mobile category when I started this discussion. 🤷
